core-dev/packages/coq/coq.8.12.dev/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "git+https://github.com/coq/coq.git"
+license: "LGPL-2.1"
```

Comment:
```suggestion
license: "LGPL-2.1-only"
```
Let's use SPDX here.

Reply:
I copied this from the `dev` package. So I should likely adjust that as well?

(Quite funny how often I get change requests for literal copies of the latest version of the opam package files. ;)

Reply:
Sure, you can adjust other packages too. Here I was just following changes made to the official packages in the general OPAM repo.

Reply:
Ah, so those got out of sync again, I see.

I adjusted the 8.11, 8.12 and dev packages. I don't think it is worth updating historic packages.